sdcm/tester.py

```diff
@@ -504,7 +504,9 @@ def argus_collect_packages(self):
             self.log.error("Unable to collect package versions for Argus - skipping...", exc_info=True)
 
     def argus_collect_manager_version(self):
-        if self.monitors.nodes and self.params.get('use_mgmt'):
+        # NOTE: K8S runs manager server in a pod on a non-monitor node.
+        #       Moreover, manager version is known in this case by the docker image tags.
+        if self.monitors.nodes and self.params.get('use_mgmt') and not self.k8s_clusters:
             manager_tool = get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
```

Comment:
so maybe instead use `scylla_manager_node` from `BaseScyllaCluster` which properly overrides mgmt node?

Reply:
It is not needed for K8S case.
Docker image tag is enough and it is available in the logs.